src/olympia/github/views.py

```diff
@@ -0,0 +1,32 @@
+import commonware
+
+from rest_framework.views import APIView
+from rest_framework.response import Response
+
+from olympia.files.models import FileUpload
+from olympia.github.tasks import process_webhook
+from olympia.github.utils import GithubRequest, GithubCallback
+
+log = commonware.log.getLogger('z.github')
+
+
+class GithubView(APIView):
+
+    def post(self, request):
+        if request.META.get('HTTP_X_GITHUB_EVENT') != 'pull_request':
+            # That's ok, we are just going to ignore it, we'll return a 2xx
+            # response so github doesn't report it as an error.
+            return Response({}, status=200)
+
+        github = GithubRequest(data=request.data)
+        if not github.is_valid():
+            return Response({}, status=422)
```

Comment:
Same here, even more here since 422 isn't really used that commonly. The constants help understanding what is returned where.